provider/letterparser_provider.py

```
@@ -0,0 +1,16 @@
+"functions shared by letterparser related activities"
+
+import os
+import docker
+from letterparser import parse
+import log
+
+IDENTITY = "process_%s" % os.getpid()
+LOGGER = log.logger("letterparser_provider.log", 'INFO', IDENTITY, loggerName=__name__)
+
+
+def docker_pandoc_output(file_name, config):
+    try:
+        return parse.docker_pandoc_output(file_name, config)
+    except docker.errors.APIError:
```

there is no possible recovery from a failure here, so remove `try`/`except` and let the exception bubble up

---

Removing leads to the error message:
```
E       docker.errors.ImageNotFound: 404 Client Error: Not Found ("no such image: knsit/pandoc:v2.5: No such image: knsit/pandoc:v2.5")
```

---

Manually pulling with `docker pull knsit/pandoc:v2.5`.

---

Run out of space since it's a 1.39GB image. We need to put the `/var/lib/docker` folder on the external volume, however here that volume has a special name `/bot-tmp` rather than `/ext`.

---

Space required is now down to 50+ MB using the alternate docker image. All green! I hope it wasn't too much work mapping the `/ext` which might not be required now the image is a more managable size?

---

This is not the first Docker image being used (also `strip-coverletter` has one plus a few that aid in testing like `sftp`); we might want to get that in nevertheless to make the Docker setup future-proof.